source/includes/inventory/troubleshooting.md

```diff
@@ -0,0 +1,7 @@
+# Troubleshooting
+
+Make sure you are always connected to the internet. If you are not in a stable environment you might see this banner on your screen. Try and reconnect to the Wifi or make sure you are in 3G/4G zone in order to resume your operations.
+
+![account](images/no-internet.png)
+
+If you encounter any problem while using the app, please send an email to : leo_at_shoprunback.com
```

Comment:
only one email per project

Reply:
I will do the same comment on the appropriate Pull Request/Repository, but If there is a bug on the application, I expect that a monitoring system is setup to handle it and report it to the team (and not the user by itself)